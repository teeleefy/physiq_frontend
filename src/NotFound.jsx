import { Card, CardBody, CardTitle, CardText } from "reactstrap";

function NotFound(){
    return(
        <Card>
            <CardTitle className="text-center">
                <h1>404</h1>
            </CardTitle>
            <CardBody>
                <h2>Sorry. Unable to locate that.</h2> 
            </CardBody>
        </Card>
    )
}

export default NotFound;