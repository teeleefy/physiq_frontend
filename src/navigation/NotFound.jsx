import { Card, CardBody, CardTitle, CardText } from "reactstrap";

function NotFound(){
    return(
        <Card>
            <CardTitle className="text-center">
                <h1>404</h1>
            </CardTitle>
            <CardBody>
                <CardText>
                   <h2>Sorry. Unable to locate that.</h2>  
                </CardText>
            </CardBody>
        </Card>
    )
}

export default NotFound;