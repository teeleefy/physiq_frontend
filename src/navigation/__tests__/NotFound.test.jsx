import { renderWithRouter, screen } from '../../../__tests__/test-tools';
import NotFound from '../NotFound'

describe('NotFound', () => {
  it('renders the NotFound component', () => {
    renderWithRouter(<NotFound />)
    screen.debug(); // prints out the jsx in the NotFound component unto the command line
  })
})

it("matches snapshot", function() {
    const { asFragment } = renderWithRouter(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });

