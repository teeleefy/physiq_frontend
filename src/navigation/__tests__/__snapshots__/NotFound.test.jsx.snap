// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="text-center card-title"
    >
      <h1>
        404
      </h1>
    </div>
    <div
      class="card-body"
    >
      <p
        class="card-text"
      />
      <h2>
        Sorry. Unable to locate that.
      </h2>
      <p />
    </div>
  </div>
</DocumentFragment>
`;
